### DISCUSSION TITLE
Unable to reset statefulset stuck in CrashLoopBackOff

### DISCUSSION THREAD

**Discussion by bar-dagan** [fluxcd/flux2#3523] [Q&A]
I have a statefulset deployed that got one of it's pods stuck in a CrashLoopBackOff. I fixed the bug in the code and uploaded a new image but while the statefulset is updated, the pod is stuck forever in CrashLoopBackOff and doesn't try to use the fixed image.
The solution offered by k8s is to manually delete the pod but I'm looking for a CD solution that will prevent these cases in clusters where I don't have direct access.

Can this issue be solved via flux?

**Comment by danbopes** [comment]
I'm having a similar issue. Were you able to get this resolved @bar-dagan ?

**Reply by bar-dagan** [reply]
No I was not
